tcms/telemetry/views.py

```diff
@@ -0,0 +1,8 @@
+from django.shortcuts import render
+from django.views.generic import TemplateView
+
+
+class TestingBreakdownView(TemplateView):
+
+    def get(self, request, *args, **kwargs):
+        return render(request, 'telemetry/testing-breakdown.html')
```

Comment:
I think you can configure an attribute `template_name` and nothing else.

FTR you can even pass this attribute as parameter to TemplateView in urls.py but it is ugly.